src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -244,14 +260,51 @@ def nlcd_soils(sjs_result):
     }
 
 
+@shared_task(throws=Exception)
+def us_groundwater_nitrogen(sjs_result):
+    """
+    Derive Groundwater Nitrogen and Phosphorus
+    """
+    if 'error' in sjs_result:
+        raise Exception('[us_groundwater_nitrogen] {}'
+                        .format(sjs_result['error']))
+
+    result = parse_sjs_result(sjs_result)
+    gwn_and_gwp_conc = groundwater_nitrogen_conc(result)
+
+    return {
+        'GrNitrConc': gwn_and_gwp_conc['groundwater_nitrogen_conc'],
```

Comment:
We decided to standardize on Python style names for these intermediate variables. Prefer `gr_ntir_conc` to `GrNitrConc`